uberfire-api/src/main/java/org/uberfire/workbench/type/ResourceTypeDefinition.java

```diff
@@ -68,4 +70,12 @@
      * @return true if matches, otherwise false
      */
     boolean accept(final Path path);
+
+    /**
+     * Defines the category for each resource. If no category is defined a default is set
+     * @return the resource category
+     */
+    default Category getCategory() {
+        return new Others();
```

Comment:
This is at least risky. Others is a managed bean and we should never create new instances.

Reply:
@ederign so I couldn't do this in there? That was the solution to not include the Others category in every class, so It was going to be the default Category for every ResourceType